docs/versions.mdx

```diff
@@ -48,7 +48,7 @@ An initial version is automatically made for new machines created when:
 
 - you import a machine from code
```

Comment:
@laurakalbag We need to remove this one for now. We had trouble with the layout so we don't create an initial version anymore for import from code.

Reply:
@kevinmaes done!